Character.cpp

```diff
@@ -87,31 +88,73 @@ int Character::takeDamage(int damage)
 }
 
 
-#include <assert>
+//#include <assert>
+
+void defeatedOther(int& attribute, std::unique_ptr<int>& initialAttribute)
+        {
+            if (attribute < *initialAttribute)
+            {
+                attribute = *initialAttribute;
+            }
+            attribute *= 1.1;
+            *initialAttribute = attribute;
+        }
```

Comment:
Please fix this indentation.
Also, this is a free function in the cpp file.  you don't need to forward declare it because it's not being used outside of this cpp file.
- [ ] revise.